### PR TITLE
[WIP] introduce smilo-commons

### PR DESCRIPTION
**Pull request by thomasmodeneis**
refactor AddResultType

### CODE REVIEW

**Comment by ElkanRoelen** [comment]
Can one of the admins verify this patch?

**Comment by ElkanRoelen** [comment]
retest this please

**Comment by ElkanRoelen** [comment]
retest this please

**Comment by ElkanRoelen** [comment]
retest this please

**Comment by elkanroelen** [comment]
retest this please

**Review by fcruz — APPROVED** [review]
all tests have passed, works fine!

File: src/test/java/io/smilo/api/AbstractSpringTest.java [hunk 1]
```diff
@@ -42,13 +42,13 @@
 public abstract class AbstractSpringTest {
 
     @Autowired
-    private TestUtility testUtility;
+    public TestUtility testUtility;
```
Comment (Krdm) [inline src/test/java/io/smilo/api/AbstractSpringTest.java]: Why did you make this public?

Comment (thomasmodeneis) [reply src/test/java/io/smilo/api/AbstractSpringTest.java]: oh it was a test but is not needed, will set back to private :)

**Pull request merged**